source/_integrations/filter.markdown

```diff
@@ -31,6 +31,7 @@ To enable Filter Sensors in your installation, add the following to your `config
 sensor:
   - platform: filter
     name: "filtered realistic humidity"
+    unique_id: sensor.realistic_humidity_filtered
```

Comment:
to avoid confusion I would not use a unique_id that resembles a entity name

Reply:
dgomes said:
> to avoid confusion I would not use a unique_id that resembles a entity name

Sounds right at first thought, but my situation was exactly the opposite problem. I was asked by some integration for an entity_id and after much searching found out that unique_id was the entity_id the other input form asked for (and I did not have it, of course). After understanding the logic behind it, I understand the reasoning as well. But I am still a newbie with HA so I also see the problem for beginners that results from the naming of this parameters.

Reply:
> unique_id was the entity_id 

unique IDs and entity IDs are not related to each other in any way. One is an internal unique identifier used in Home Assistant registries, while the entity ID is an identifier for end-user consumption.